server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -841,10 +844,47 @@ exports.redirectTopologyChangeInformation = function (body, user, originator, xC
         customerJourney
       );
 
-      resolve();
+  let forwrdingContructResponse = await controlConstruct.getForwardingDomainListAsync()
+  let serverclientinterfacepac;
+  let logicalterminationpoint = await fileOperation.readFromDatabaseAsync(
+    onfPaths.LOGICAL_TERMINATION_POINT
+  );
+  for (let i = 0; i < logicalterminationpoint.length; i++) {
+    let layerprotocol = logicalterminationpoint[i][onfAttributes.LOGICAL_TERMINATION_POINT.LAYER_PROTOCOL]
+    for (let j = 0; j < layerprotocol.length; j++) {
+   let layerProtocalName = layerprotocol[j]['layer-protocol-name']
+   if(layerProtocalName == 'operation-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_OPERATION_LAYER'){
+    let operationclientinterfacepac = layerprotocol[j][onfAttributes.LAYER_PROTOCOL.OPERATION_CLIENT_INTERFACE_PAC]
+    let clientconfiguration = operationclientinterfacepac[onfAttributes.OPERATION_CLIENT.CONFIGURATION]
+  if(clientconfiguration  !== undefined){
+   delete clientconfiguration['operation-key'];
+   }
+  }
+   else if(layerProtocalName == 'operation-server-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_OPERATION_LAYER'){
+   serverclientinterfacepac = layerprotocol[j][onfAttributes.LAYER_PROTOCOL.OPERATION_SERVER_INTERFACE_PAC]
+        let serverconfiguration = serverclientinterfacepac[onfAttributes. OPERATION_SERVER.CONFIGURATION]
+       if(serverconfiguration  !== undefined){
+       delete serverconfiguration['operation-key'];
+       }
+       }
+      }
+    }
+
+     let controlConstructResponse = {
+        "core-model-1-4:control-construct ":forwrdingContructResponse," logical-termination-point": logicalterminationpoint
+      };
+     /****************************************************************************************
+       * Setting 'application/json' response body
+       ****************************************************************************************/
+      response['application/json'] = controlConstructResponse;
```

Comment:
The resulting response JSON format is as below which is wrong , 
```
{
  'core-model-1-4:control-construct ': [ 
          { 
              'uuid': 'ro-0-0-1-op-fd-0000', 
              'forwarding-construct': [Array] 
          } 
    ],  
   ' logical-termination-point': [Array] 
}
```

1. core-model-1-4:control-construct should be an object not an array 
2. forwarding-construct should not be under directly to core-model-1-4:control-construct. The correct hierarchy is   core-model-1-4:control-construct/forwarding-domain=<uuid>/forwarding-construct
3. logical-termination-point should be within core-model-1-4:control-construct object
4. Could you please correct the format as below , 

```
{
  'core-model-1-4:control-construct ': 
          { 
              'uuid': 'ro-0-0-1-op-fd-0000', 
              'logical-termination-point': [Array] 
              'forwarding-domain': [Array] 
          } 
}
```
Many thanks.
Also , please refer the load file for a reference.